_posts/2017-10-05-denounce-segwit2x.md

```diff
@@ -10,7 +10,13 @@ date: 2017-10-05
 ---
 On 2017-10-11 at noon (UTC), Bitcoin.org is planning to publish a banner on every page of the site warning users about the risks of using services that will default to the so-called Segwit2x (S2X) contentious hard fork. S2X companies will be called out by name. To ensure that we only warn users against companies that will actually put user deposits at risk, we urge all companies to publicly clarify their stance before the above date, either by a highly-visible public statement or by commenting on Bitcoin.org issue #[1835](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1835) (or by doing both).
 
-If there are mitigating factors, such as your company planning to continue supporting Bitcoin by default and only supporting S2X insofar as required to allow users to sell or withdraw their S2X coins, please inform us of these details on the above issue so we can remove your entry if appropriate.
+In particular, we need to know that:
+
+1. The company will not under any circumstances list "Segwit2x" as "BTC" and/or "Bitcoin". Note that [Bitcoin is not ruled by miners](https://en.bitcoin.it/wiki/Bitcoin_is_not_ruled_by_miners), and miner actions cannot be used as a justification to redefine Bitcoin.
+2. The company will not by default *convert* user BTC to S2X-coins. Distributing "air-dropped" S2X-coins to users is acceptable, however.
```

Comment:
Can you clarify what is meant by "convert"? Do you mean switching to using S2X by default or selling all user BTC for S2X on an exchange somewhere?